<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureAudio</class>
 <widget class="QWidget" name="ConfigureAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>332</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Audio</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_emulation">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_emulation">
          <property name="text">
           <string>Emulation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="emulation_combo_box"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>Output Engine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="output_sink_combo_box"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="toggle_audio_stretching">
        <property name="toolTip">
         <string>This post-processing effect adjusts audio speed to match emulation speed and helps prevent audio stutter. This however increases audio latency.</string>
        </property>
        <property name="text">
         <string>Enable audio stretching</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label2">
          <property name="text">
           <string>Audio Device</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="audio_device_combo_box"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Volume:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="volume_slider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="volume_indicator">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0 %</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Microphone</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Input Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_type_combo_box">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Real Device</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Input Device</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_device_combo_box"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>167</width>
       <height>55</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
